README.md

```diff
@@ -52,7 +52,7 @@ Resources
 <!-- NOTE!!! This is copied from SUPPORT.md.  If you update it here, update it there as well.) -->
 -   Converse with us in our CORTX-Open Source Slack channel [![Slack](https://img.shields.io/badge/chat-on%20Slack-blue")](https://cortx.link/slack_invite) to interact with community members and gets your questions answered.
 -   Join us in [Discussions](https://github.com/Seagate/cortx/discussions) to ask, answer, and discuss topics with your fellow CORTX contributors.
--   Check out our [Youtube Channel](https://cortx.link/XGVHVq).
+-   Check out our [Youtube Channel](https://www.youtube.com/playlist?list=PLOLUar3XSz2M_w5OxQLNDBTpSrsGbjDWr).
```

Comment:
Why are you changing this?  Did you test the link?  @novium258 changed this link so we can get some tracking metrics.  If the link works, we shouldn't change it back.  :)